circuits/main/src/get_account.nr

```diff
@@ -80,27 +80,21 @@ pub(crate) fn assert_account_equals(account_rlp_left_padded: [u8; MAX_ACCOUNT_ST
     );
 }
 
-fn assert_account_proof(account_with_proof: AccountWithProof, state_root: [u8; KEY_LENGTH]) {
+fn assert_account_proof(account_state_proof: StateProof, state_root: [u8; KEY_LENGTH]) {
     let trie_proof = TrieProof {
-        key: account_with_proof.key,
-        proof: account_with_proof.proof,
-        depth: account_with_proof.depth,
-        value: account_with_proof.value
+        key: account_state_proof.key,
+        proof: account_state_proof.proof,
+        depth: account_state_proof.depth,
+        value: account_state_proof.value
     };
     assert(trie_proof.verify_state_root(state_root), "TrieProof: Invalid state root");
 }
 
 pub(crate) fn verify_account(block_no: Field, address: [u8; 20], state_root: [u8; KEY_LENGTH]) -> Account {
```

Comment:
This is outside of this PR diff but I think that:
1. If this is library API method this should be called `get_account`
2. It should have visibility `pub` instead of `pub(crate)`

Reply:
`pub get_account` method will be a user-facing method that will do more than this method. It will verify the account against block hash and use block hash oracle under the hood. This method is our utility method to do only account-relater stuff. It proves up to a state_root. Therefore it's pub(crate)